README.md

```diff
@@ -14,19 +15,78 @@ Takeaway Challenge
 
  ```
 
-Instructions
+
+My solution to the Makers Academy Week 2 challenge, building a Ruby app for a restaurant in which a user can place orders. This was an exercise in TDD, including the extensive use of mocking to prevent class and API dependencies being hardcoded, as well as OOP and some basic Polymorphism.  It was also my first experience using the Twilio API.
+
+My Implementation
 -------
+I broke the user stories down into several interacting classes.
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+ - **Restaurant**: the main object that holds all others together.
+ - **OrderHistory**: a list of orders with the capability to generate new orders.
+ - **Order**: a list of food, and associated methods.
+ - **Good**: a parent class to **Food** and **Drink**, done to make the classes as universally applicable as possible for the projects longevity.
 
-Task
------
+All unit testing has been completed using mocking to ensure that the code would be easy to change, as well as to test Twilio without making a real API call. I've then implemented a demonstration of the classes working together in the './demo.rb' file.
+
+The idea was to make everything as modular and customisable as possible, to ensure that you could use this code as a jumping off point for a platform hosting multiple restaurants and serving multiple customers.
+
+Installing the app
+-------
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
+ - Clone the Github repo
+ - Register for a free Twilio account
+ - In the parent directory, create a file called .env
+ - Add to this file your Twilio API key, token, to and from numbers as outlined here:
+         
+       export TWILIO_ACCOUNT_SID='XXXXXXXXXXXXXXX'
+       export TWILIO_AUTH_TOKEN='XXXXXXXXXXXXXXX'
+       export TWILIO_TO=+44XXXXXXXXX
+       export TWILIO_FROM=+44XXXXXXXXX
+       
```

Comment:
Useful explanation for the user on how to use ENV VARs for the Twilio REST API functionality.